components/feature/recentlyclosed/src/main/java/mozilla/components/feature/recentlyclosed/RecentlyClosedTabsStorage.kt

```diff
@@ -25,11 +25,11 @@ import java.io.File
  * A storage implementation that saves snapshots of recently closed tabs / sessions.
  */
 @Suppress("TooManyFunctions")
-class RecentlyClosedTabsStorage(
+internal class RecentlyClosedTabsStorage(
```

Comment:
Can this be a breaking change worthy of being in the change log?

Reply:
In theory yes, but I looked around and couldn't find anyone creating a custom instance - which was also never the intention.